torchdynamo/variables/tensor.py

```diff
@@ -70,6 +70,8 @@ def create(cls, tx, proxy, example_value=None, nnmodule=None, **options):
 
         if isinstance(example_value, torch.Tensor):
             proxy.node.meta["example_value"] = example_value.clone()
+            if example_value.grad is not None:
+                proxy.node.meta["example_grad_value"] = example_value.grad.clone()
```

Comment:
Thoughts on `"example_grad_value"` versus `meta["example_value"].grad` ?

The latter might propagate through some PyTorch ops to other example values, so I'd lean towards that -- but perhaps there are pros of this one I havent thought of?

Reply:
I like the `.grad` (latter) version better too. Made the changes.